src/Steps/Release/PublishRelease.php

```diff
@@ -150,7 +134,9 @@ protected function stabiliseRequirements(OutputInterface $output, LibraryRelease
 
         // Save modifications to the composer.json for this module
         if ($composerData !== $originalData) {
-            $this->updateComposerData($output, $parentLibrary, $composerData);
+            $parentName = $parentLibrary->getName();
+            $this->log($output, "Rewriting composer.json for <info>$parentName</info>");
+            $parentLibrary->setComposerData($composerData, true, 'MNT Update release dependencies');
```

Comment:
```suggestion
            $parentLibrary->setComposerData($composerData, true, 'DEP Update release dependencies');
```
I'd the stabilise commit should have a 'DEP' and the dev commit should have a MNT

Reply:
The stabilise commits _already_ has `MNT` - this PR isn't going to change those. If you think those should change, you can create a separate card to handle that unrelated change.